### ISSUE TITLE
collection node delegate methods for supplementary Element displaying are never called

### ISSUE THREAD

**Issue by shellhue** [TextureGroup/Texture#885]
I find that ASCollectionNode delegate method `collectionNode:willDisplaySupplementaryElementWithNode:` and `collectionNode:didEndDisplayingSupplementaryElementWithNode:` are never called.

Is there any other configuration to trigger these method to be called? Or it is a bug?

**Comment by bimawa** [comment]
Yes its a bug, but it's fixed, check this commit: https://github.com/TextureGroup/Texture/commit/eab7bea48de9ceb8b57ff96f6ec5f9f827800384
